pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_about-xrefs-validation-in-pantheon.adoc

```diff
@@ -0,0 +1,46 @@
+[id="about-xrefs-validation-in-pantheon_{context}"]
+= About xref validation in Pantheon
+
+[role="_abstract"]
+When you upload a Git repository to {ProductShortName}, each module and assembly in the repository is built as an html document. During this build, {ProductShortName} checks whether xrefs in each module and assembly are valid.
+
+If {ProductShortName} finds invalid xrefs in a module or assembly, a red alert icon is displayed alongside it in the Search Page.
+
+Only the following forms of xref are validated:
+ 
+---- 
+xref:document-b.adoc#target-section[Target section]
+---- 
+
+----
+xref:../modules/document-c.adoc[Content C]
+----
+
+[discrete]
+== Example of using xref in {ProductShortName}
+
+In the this example, the following are the source and targets:
+
+source document:: `document-a.adoc` 
+targets::
++
+** A section called `target section` in a document `document-b.adoc.`
+** A document `document-c.adoc`.
+
+Use xrefs as follows:
+
+* To cross-reference section `target section` in `document-b.adoc` from `document-a.adoc`, use the following form:
++
+----
+xref:document-b.adoc#target-section[target section] 
```

Comment:
what if repo A and repo B both have a file named `document-b.adoc`? Do xrefs only work within a given repo? Or are all modules visible to all other modules within Panteon, regardless of the source repo?

Reply:
@stoobie atm, xrefs only work within the same repo. 
So, if repo and A and B have a file document-b.adoc and the xref in repo A will point to the document-b.adoc in repo A and xref in repo B will point to the file document-b.adoc in repo B